community/kernel/src/main/java/org/neo4j/kernel/impl/api/StateHandlingStatementOperations.java

```diff
@@ -1475,4 +1475,32 @@ public String relationshipLegacyIndexRemoveConfiguration( KernelStatement statem
         return legacyIndexStore.getAllRelationshipIndexNames();
     }
     // </Legacy index>
+
+
+    private NodeCursor acquireSingleNodeCursor( KernelStatement statement, long nodeId )
+    {
+        NodeCursor cursor = statement.getStoreStatement().acquireSingleNodeCursor( nodeId );
+        if ( statement.hasTxStateWithChanges() )
+        {
+            return statement.txState().augmentSingleNodeCursor( cursor );
+        }
+        else
+        {
+            return cursor;
+        }
+    }
+
+    private RelationshipCursor acquireSingleRelationshipCursor( KernelStatement statement, long relationshipId )
+    {
+        RelationshipCursor cursor = statement.getStoreStatement().acquireSingleRelationshipCursor( relationshipId );
+        if ( statement.hasTxStateWithChanges() )
+        {
+            return statement.txState().augmentSingleRelationshipCursor( cursor );
+        }
+        else
+        {
+            return cursor;
+        }
+
```

Comment:
There's something about empty lines like these that makes my skin crawl... sorry